openforcefield/tests/test_examples.py

```diff
@@ -82,7 +85,7 @@ def find_examples():
 
     example_file_paths = []
     for example_file_path in glob.glob(os.path.join(examples_dir_path, '*', '*.py')):
-        example_file_path = os.path.relpath(example_file_path)
+        example_file_path = os.path.abspath(example_file_path)
```

Comment:
I agree this is a good point to refactor. A few notes:

* Globbing for something ending in `*.py` ignores all notebooks, and only catches a single file.

```
ipdb> [*examples_dir_path.glob('*/*.py')]
[PosixPath('/Users/mwt/software/openforcefield/examples/conformer_energies/conformer_energies.py')]
```
* Are the notebooks tested in CI? Will move this into an issue if appropriate
* Likewise, no "slow" examples are being run, so the likelihood that the existing logic is broken is hidden from us. I'd prefer to either skip this logic altogether or forcefully add a "slow" example back in. The logic I have in place for skipping examples that require RDKit should be transferrable to other criteria

Reply:
A few updates

* Notebooks were _not_ being tested ... since #561. But they were before. This got introduced since the notebooks were tested implicitly beforehand and it was not clear in the process of that PR that they were skipped after those changes.
  * `pytest --nbval-lax --other-args openforcefield/tests` will collect only the tests in that directory
  *  `pytest --nbval-lax --other-args` will see the various `examples/*/*ipynb` files since they are accessible in the root directory.
  * In either case, globbing `examples/*/*.py` will only see Python scripts, not Jupyter notebooks.

* Independently, pytest bumped to 6.0.0 last night and broke nbval (and perhaps other things). I'll pin pytest<6 for now, but it looks like the fix is already in place https://github.com/computationalmodelling/nbval/pull/152. Hopefully, this pin does not last long.

Reply:
Above  PR was merged, released to PyPI, and should be on conda-forge soon https://github.com/conda-forge/nbval-feedstock/pull/10. Builds will be broken until then.